.gitignore

```diff
@@ -1,3 +1,6 @@
 *.pyc
```

Comment:
@rddaz2013 Can you create  a global gitignore file on windows? I'll use those on linux to keep emacs specific configurations out of the source tree